util/topsql/reporter/datamodel.go

```diff
@@ -497,10 +497,46 @@ func (c *collecting) appendOthersStmtStatsItem(timestamp uint64, item stmtstats.
 	others.appendStmtStatsItem(timestamp, item)
 }
 
+// removeInValidPlanRecord remove "" plan if there are only 1 valid plan in the record.
+func (c *collecting) removeInValidPlanRecord() {
```

Comment:
```suggestion
func compactPlans(records) records {
```

Reply:
`compact` sounds meanless to the user, he still needs to read the comment or code...